integration-tests/gradle/projects/it-basic-groovy/build.gradle

```diff
@@ -19,6 +19,23 @@ dokkaHtml {
             displayName.set("custom")
             reportUndocumented.set(true)
         }
+
+        configureEach {
+            perPackageOption { // testing closures
+                matchingRegex.set(".*internal.*")
+                suppress.set(true)
+            }
+
+            sourceLink { // testing closures
+                localDirectory.set(file("src/main"))
+                remoteUrl.set(
+                        new URL(
+                                "https://github.com/Kotlin/dokka/tree/master/" +
+                                        "integration-tests/gradle/projects/it-basic-groovy/src/main"
+                        )
+                )
+            }
+        }
```

Comment:
Added this in advance to test groovy-based configuration. It works now and will fail if closures are removed and no replacement is in place.